### PR TITLE
fix(ONYX-301): padding is wrong on the follows page, collected artists rail and myc empty state

### PR DESCRIPTION
**Pull request by dariakoko**
This PR resolves [ONYX-301] 

### Description

| Artist collected rail Before | Artist collected rail After | Artist collected rail Before | Artist collected rail After |
| --- | --- | --- | --- |
|  |  |  |  |

| My Collection Empty State Before | My Collection Empty State After | 
| --- | --- |
|  |  |

| Follows tab Before | Follows tab After |
| --- | --- |
|  |  |
 

### PR Checklist

- [x] I have tested my changes on **iOS** and **Android**.
- [ ] I hid my changes behind a **[feature flag]**, or they don't need one.
- [ ] I have included **screenshots** or **videos**, or I have not changed the UI.
- [ ] I have added **tests**, or my changes don't require any.
- [ ] I added an **[app state migration]**, or my changes do not require one.
- [ ] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [ ] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- fix padding on the follows page, collected artists rail and myc empty state - daria

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[ONYX-301]: https://artsyproduct.atlassian.net/browse/ONYX-301?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/Scenes/MyCollection/Components/MyCollectionCollectedArtistsRail.tsx [hunk 1]
```diff
@@ -57,8 +57,6 @@ export const MyCollectionCollectedArtistsRail: React.FC<MyCollectionCollectedArt
 
   return (
     <Flex testID="my-collection-collected-artists-rail">
-      <Flex position="absolute" top={0} left={0} right={0} height={1} bg="black5"></Flex>
-
```
Comment (dariakoko) [inline src/app/Scenes/MyCollection/Components/MyCollectionCollectedArtistsRail.tsx:61]: Not sure why this component was here, it's not in the design

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Cross-platform user-facing changes (fix padding on the follows page, collected artists rail and myc empty state - daria)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 0f3222e0c9438bbd29af4cd9489d77bdd0260afa
</p>

**Review by olerichter00 — DISMISSED** [review]
Thanks a lot for the fix 🙇

File: src/app/Scenes/MyCollection/Components/MyCollectionZeroStateArtworks.tsx [hunk 2]
```diff
@@ -13,22 +13,23 @@ export const MyCollectionZeroStateArtworks: React.FC = () => {
...
+                source: Tab.collection,
+              },
+            })
+          }}
+          variant="outline"
+          size="small"
+        >
+          Add Artworks
+        </Button>
+      </Flex>
```
Comment (dariakoko) [inline src/app/Scenes/MyCollection/Components/MyCollectionZeroStateArtworks.tsx:32]: The clickable area of the button was on the full screen's width. Wrapping it with `Flex` fixes it

**Pull request merged**